config/mysqlpersister/mysqlpersister.go

```diff
@@ -164,18 +185,20 @@ func (mp *MysqlPersister) PersistAndNotify(_ string, c *qsc.ServiceConfig) error
 
 	_, err = mp.db.Exec(q, args...)
 	if err != nil {
-		mysqlErr, ok := err.(*mysql.MySQLError)
-		if ok && mysqlErr.Number == 1062 {
+		if mysqlErr, ok := err.(*mysql.MySQLError); ok && mysqlErr.Number == 1062 {
```

Comment:
nit: I think it would be neat to make `1062` a const with a descriptive name.

Reply:
Good idea. Did it